python/ray/tests/test_object_spilling.py

```diff
@@ -316,6 +315,33 @@ def test_spill_objects_automatically(object_spilling_config, shutdown_only):
     assert_no_thrashing(address["redis_address"])
 
 
+# @pytest.mark.skipif(
```

Comment:
I tested this by throw exception in writing. I'm still trying to figure out how to test it. I tried mock but it doesn't work because the code will be invoked in another process.